idea_town/experiments/views.py

```diff
@@ -14,7 +14,7 @@ class ExperimentViewSet(viewsets.ModelViewSet):
     Returns a list of all Experiments in the system.
     """
     queryset = Experiment.objects.all()
-    serializer_class = ExperimentSerializer
+    serializer_class = ExperimentDeepSerializer
```

Comment:
@lmorchard Yeah, I could have explained this change more clearly: it keeps the front-end code super simple if we pull down the entire experiment when we fetch the collection at app startup. I see what you're saying about DB and API efficiency, but we're going to launch with at most 3 experiments, and the lineup isn't going to change often: there will be an Idea Town committee that signs off on changes.

So, the DB join result is going to be eminently cacheable (either inside postgres or inside django's ORM), and I think deeper performance enhancements are probably premature optimizations at this point.

If you feel strongly about keeping the deep and shallow serializers, I'm happy to back out these python changes, and update the front-end code to separately fetch the model details when the details page is loaded.